### DISCUSSION TITLE
Pre-PPC submitted

### DISCUSSION THREAD

**Discussion by Ovid** [Perl-Oshun/oshun#49] [Announcements]
Hi all.

Perhaps it's jumping the gun, but given much of the bike-shedding and some contentious P5P discussion earlier this year, I got burnt out. I decided to bite the bullet and send a pre-PPC email for data checks to P5P: https://www.nntp.perl.org/group/perl.perl5.porters/2024/01/msg267600.html

Best,
Ovid

**Comment by pders01** [comment]
Looks like Christmas is coming early this year 🎄
